frontend/src/scenes/project/Settings/GroupAnalytics.tsx

```diff
@@ -85,6 +85,7 @@ export function GroupAnalytics(): JSX.Element | null {
                     Cancel
                 </LemonButton>
             </div>
+            <LemonDivider className="my-6" />
```

Comment:
This is a little bug fix where you could end up with 2 dividers in a row if group analytics wasn't enabled.